Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -303,30 +303,42 @@ theorem lsub_lt_ord {ι} {f : ι → Ordinal} {c : Ordinal} (hι : #ι < c.cof)
     (∀ i, f i < c) → lsub.{u, u} f < c :=
   lsub_lt_ord_lift (by rwa [(#ι).lift_id])
 
-theorem cof_sup_le_lift {ι} {f : ι → Ordinal} (H : ∀ i, f i < sup.{u, v} f) :
-    cof (sup.{u, v} f) ≤ Cardinal.lift.{v, u} #ι := by
+theorem cof_iSup_le_lift {ι} {f : ι → Ordinal} (H : ∀ i, f i < iSup f) :
+    cof (iSup f) ≤ Cardinal.lift.{v, u} #ι := by
   rw [← sup_eq_lsub_iff_lt_sup.{u, v}] at H
   rw [H]
   exact cof_lsub_le_lift f
 
-theorem cof_sup_le {ι} {f : ι → Ordinal} (H : ∀ i, f i < sup.{u, u} f) :
-    cof (sup.{u, u} f) ≤ #ι := by
+@[deprecated cof_iSup_le_lift (since := "2024-08-27")]
+alias cof_sup_le_lift := cof_iSup_le_lift
+
+theorem cof_iSup_le {ι} {f : ι → Ordinal} (H : ∀ i, f i < iSup f) :
+    cof (iSup f) ≤ #ι := by
   rw [← (#ι).lift_id]
-  exact cof_sup_le_lift H
+  exact cof_iSup_le_lift H
+
+@[deprecated cof_iSup_le (since := "2024-08-27")]
+alias cof_sup_le := cof_iSup_le
```

Comment:
Why are you changing the statement of this lemma? Please keep it the same, then it won't show up in the diff. Same everywhere else

Reply:
What do you mean? All I changed was `sup` to `iSup`.

Reply:
Does this also apply to all the stuff I didn't rename, like all the stuff about `bsup`? Or should I rename that stuff too, even though it'll probably make the diff like 2x larger, and I'm probably deprecating all of that soon anyways?

Reply:
Or I guess there's the third option, which is to disable the deprecation linters on those sections of the file.

Reply:
I disabled the deprecation linter on all the basic API for `bsup`, `lsub`, `blsub`, and `mex`. I also left TODO notes about what should be done for each.